installer/install.iss

```diff
@@ -1730,6 +1754,10 @@ begin
         else
             RdbSSH[GS_OpenSSH].Checked:=True;
         end;
+
+        data:=ReplayChoice('Tortoise Option','');
+        if (data='true') then
+            TortoisePlink.Checked:=True;
```

Comment:
In the surrounding code, we don't seem to call `ReplacyChoice()`. Where do the other pages call that function?

Reply:
This is immediately after the replay for the radio buttons starting at line 1750.

Reply:
Ah, right, there were no checkboxes to record previously. Thanks for clarifying!